### ISSUE TITLE
Sharing links from browser to Zim wiki notes 

### ISSUE THREAD

**Issue by Self-Perfection** [gsantner/markor#2409]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**. Not a [question or feature request](https://github.com/gsantner/markor/issues/new/choose).
- [X] The topic is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=#js-issues-search). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.
- [X] The bug is still present in the latest development version (git master). (Please [download](https://nightly.link/gsantner/markor/workflows/build-android-project/master) and try the test version of Markor, named **Marder**. Don't worry; Markor and Marder appear as completely separate applications. You can install both side-by-side, and Markor settings are not touched. In case the issue is resolved there, you don't need to create a bug report. The change will be part of the next Markor update.)

### Description

Shared links from web browser are lost.

### Steps to reproduce

1.  Open https://news.ycombinator.com/item?id=35618503 in browser
2. Share link to Markor
3. Do not edit proposed text and choose one of existing Zim wiki notes

For a long you would get

```
----
[Lightweight SQLite Editor for Windows / Hacker News](https://news.ycombinator.com/item?id=35618503 )
```

This is not a Zim wiki syntax but is generally acceptable. A couple of month ago there were a regression and now I get

```
----
[[LINK|TITLE]]
```

All shared information is lost!
The best result would be get a link in Zim format:
```
[[https://news.ycombinator.com/item?id=35618503|Lightweight SQLite Editor for Windows / Hacker News]]
```

but actually I would be happy if sharing will retain actual title and URL regardless of format.

### Information

Android version: CalyxOS 14
Device: Irrelevant
App Version: 2.12.5

### Source

F-Droid

### Format / File type

Wikitext, Zim

### Additional info / Log

```shell
-
```

**Comment by harshad1** [comment]
This is a bug. I will fix it.

**Comment by harshad1** [comment]
Solved in #2377
